src/components/Counter/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Text, TouchableHighlight, View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+import { styles } from '../../styles'
+import * as actions from './actions'
+import * as selectors from './reducer'
+
+export const Counter = () => {
+  const dispatch = useDispatch()
+  const amountOfPullUps = useSelector(selectors.getPullUps)
```

Comment:
I think the current amount of pull ups could also be stored via the `useState` hook and not in the store. If one saves the current progress than the amount is saved in the store as `totalPullUps´